src/components/formElements/TextArea.tsx

```diff
@@ -0,0 +1,22 @@
+import { forwardRef, ForwardRefRenderFunction } from "react";
+import { TextAreaProps } from "./FormElements.types.ts";
+import { TextAreaStyled } from "./FormElements.styles.ts";
+
+const ForwardRefTextAreaField: ForwardRefRenderFunction<
+  HTMLTextAreaElement,
+  TextAreaProps
+> = ({ placeholder, accept, ...props }, forwardedRef) => {
```

Comment:
what's `accept`? the prop name doesn't tell much. I'd say it's some sort of callback judging by name but I don't know what it does

Reply:
my mistake. It was for input